### PR TITLE
refactor(lsp): join help and completion

### PR DESCRIPTION
**Pull request by gak**
This should make it easier to add new completions.

### CODE REVIEW

**Review by wesbillman — APPROVED** [review]
Nice! Great idea

File: lsp/completion.go [hunk 1]
```diff
@@ -18,68 +18,35 @@ var enumTypeCompletionDocs string
...
-func ${3:Name}(ctx context.Context, req ${1:Request}) (${2:Response}, error) {
-	return ${2:Response}{}, nil
-}`),
-		Documentation: &protocol.MarkupContent{
-			Kind:  protocol.MarkupKindMarkdown,
-			Value: verbCompletionDocs,
-		},
-		InsertTextFormat: &insertTextFormat,
-	}
+var completionItems = []protocol.CompletionItem{
```
Comment (alecthomas) [inline lsp/completion.go:21]: Nice idea, but there should be a comment somewhere in the code describing the format.

Comment (gak) [reply lsp/completion.go:21]: https://github.com/TBD54566975/ftl/pull/1776/files#diff-596afc4cb243f626e08e3f9a1e6b673138da74be0747e3c67fa652e4e765d5d8R32 ?

I can write more on it if you want.

Comment (alecthomas) [reply lsp/completion.go:21]: Yeah I saw that, but it's mid-code and not really obvious to future engineers wanting to add more snippets. Maybe just move that? Drop it in the next PR, no need for a dedicated one.

**Pull request merged**